### PR TITLE
meta(oracle): add oracle PR scope

### PR DESCRIPTION
**Pull request by WikiRik**
## Pull Request Checklist

- [X] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [X] Does the name of your PR follow [our conventions](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md#6-commit-your-modifications)?

## Description Of Change

Adds the Oracle scope to the possible options since support for it has landed in v6.

### CODE REVIEW

File: .github/workflows/pr-title.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ jobs:
             mariadb
             mssql
             mysql
+            oracle
```
Comment (WikiRik) [inline .github/workflows/pr-title.yml:28]: I was wondering if we want to use oracle or oracledb here. Feel free to let me know your ideas on it

**Comment by fzn0x** [comment]
Since v7 will be landed soon I think it's a good addition, they will be added the scope for the v7 PR

File: .github/workflows/pr-title.yml (hunk 1 above)
Comment (fzn0x) [reply .github/workflows/pr-title.yml:28]: > I was wondering if we want to use oracle or oracledb here. Feel free to let me know your ideas on it

oracle is fine for me

**Review by fzn0x — APPROVED** [review]

File: .github/workflows/pr-title.yml (hunk 1 above)
Comment (ephys) [reply .github/workflows/pr-title.yml:28]: I vote to always follow whatever is written here https://github.com/sequelize/sequelize/blob/v6/src/sequelize.d.ts#L172

Comment (WikiRik) [reply .github/workflows/pr-title.yml:28]: Makes sense to me!

**Comment by ephys** [comment]
> Since v7 will be landed soon

😶‍🌫️

**Pull request merged**